packages/perseus-editor/src/components/graph-locked-figures/locked-polygon-settings.tsx

```diff
@@ -54,6 +58,31 @@ const LockedPolygonSettings = (props: Props) => {
         onChangeProps({color: newValue});
     }
 
+    function handlePolygonMove(movement: "up" | "down" | "left" | "right") {
```

Comment:
[Non-blocking, newbie question]
This is a cool way to specify what string values the function should accept! Is this strategy preferred over an enum? (Maybe for the sake of simplicity, since it's only used once?) Thanks:)

Reply:
As far as I can tell, enums and union types like this pretty much serve the same function. I do it like this since it's more consistent with the rest of the codebase. I'd be curious to hear what other people on the Perseus team think though.